tests/pluto/Pluto.test.ts

```diff
@@ -400,12 +389,12 @@ describe("Pluto", () => {
     const receiver = Domain.DID.fromString("did:prism:321");
     const name = "test";
 
-    const privateKey: Domain.PrivateKey = new Ed25519PrivateKey(
+    await instance.storePrismDID(host, new Ed25519PrivateKey(
       Buffer.from("01011010011101010100011000100010")
-    );
-
-    await instance.storePrismDID(host, privateKey);
-    await instance.storePrismDID(receiver, privateKey);
+    ));
+    await instance.storePrismDID(receiver, new Ed25519PrivateKey(
+      Buffer.from("01011010011101010100011000100010")
+    ));
```

Comment:
Because we cannot insert duplicate entries on privateKey. We instantiated privateKey + where generating privateKeys with the same ID. 

The inMemory storage didn't throw error when duplicate ID or rows are added. That is the problem, so each time i need to use a different key, i'm using a new instance.